doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -415,6 +415,20 @@ Field       | Type       | Meaning
 `worker_id` | [`bigint`] | The ID of the worker thread hosting the dataflow.
 `time`      | [`mz_timestamp`] | The next timestamp at which the dataflow may change.
 
+### `mz_storage_host_metrics`
+
+The `mz_storage_host_metrics` table gives the last known CPU and RAM utilization statistics
+for all processes of all extant storage replicas.
+
+At this time, we do not make any guarantees about the exactness or freshness of these numbers.
+
+Field              | Type       | Meaning
+-------------------|------------|--------
+`id`               | [`text`]   | The ID of the storage object (source or sink).
+`process_id`       | [`bigint`] | An identifier of a process within a replica.
```

Comment:
```suggestion
`process_id`       | [`bigint`] | An identifier of a process within a sink or source. Larger `SIZE`'s will have more than 1 process.
```

Reply:
> Larger `SIZE`'s will have more than 1 process.

We are intentionally not documenting this fact!